### PR TITLE
Tf-TRT minor test fixes

### PR DESCRIPTION
**Pull request by pooyadavoodi**
- Check if checkpoints are available in TF-TRT quantization test
- Use constant input test tensors instead of random tensors Otherwise the test sometimes fails depending on the generated random values.

### CODE REVIEW

File: tensorflow/python/compiler/tensorrt/test/tf_trt_integration_test_base.py [hunk 1]
```diff
@@ -315,7 +315,10 @@ def _RunGraph(self,
             for expected_shape, actual_val in zip(expected_shapes, new_val):
               self.assertEqual(list(expected_shape), list(actual_val.shape))
             if val is not None:
-              self.assertAllClose(val, new_val, atol=1.e-06, rtol=1.e-06)
+              self.assertAllClose(
```
Comment (aaroey) [inline tensorflow/python/compiler/tensorrt/test/tf_trt_integration_test_base.py]: As chatted offline, please revert the changes here or just apply it to specific test (if applicable) instead.
Thanks.

Comment (pooyadavoodi) [reply tensorflow/python/compiler/tensorrt/test/tf_trt_integration_test_base.py]: @aaroey I reverted this change.

Comment (pooyadavoodi) [reply tensorflow/python/compiler/tensorrt/test/tf_trt_integration_test_base.py]: Though a similar change was made recently:

```
              # Some ops may have nondeterministic output. E.g. Conv2D may use
              # winograd algorithm. So we set atol/rtol be larger than 1.e-06.
              self.assertAllClose(val, new_val, atol=1.e-05, rtol=1.e-05)
```

**Review by aaroey — DISMISSED** [review]

**Comment by aaroey** [comment]
Hi @pooyadavoodi, would you please fix this error?
```
tensorflow/python/compiler/tensorrt/test/const_broadcast_test.py:21: [W0611(unused-import), ] Unused numpy imported as np
```

**Comment by pooyadavoodi** [comment]
> Hi @pooyadavoodi, would you please fix this error?
> 
> ```
> tensorflow/python/compiler/tensorrt/test/const_broadcast_test.py:21: [W0611(unused-import), ] Unused numpy imported as np
> ```

Done

**Review by aaroey — DISMISSED** [review]

**Review by aaroey — APPROVED** [review]

**Comment by aaroey** [comment]
> * Increase tolerance of TF-TRT tests for fp32 and fp16
> * Check if checkpoints are available in TF-TRT quantization test
> * Use constant input test tensors instead of random tensors Otherwise the test sometimes fails depending on the generated
> * random values.

@pooyadavoodi could you please update the description?

**Comment by pooyadavoodi** [comment]
> > * Increase tolerance of TF-TRT tests for fp32 and fp16
> > * Check if checkpoints are available in TF-TRT quantization test
> > * Use constant input test tensors instead of random tensors Otherwise the test sometimes fails depending on the generated
> > * random values.
> 
> @pooyadavoodi could you please update the description?

Done
